stl/inc/yvals_core.h

```diff
@@ -793,6 +793,7 @@
 // warning C4702: unreachable code
 // warning C4793: function compiled as native
 // warning C4820: 'N' bytes padding added after data member 'meow' (/Wall)
+// warning C4868: compiler may not enforce left-to-right evaluation order in braced initializer list

```

Comment:
These comments say "(/Wall)" for off-by-default warnings to remind us of their nature, so we should follow that pattern here.